src/View/Helper/UrlHelper.php

```diff
@@ -195,11 +218,14 @@ protected function _encodeUrl($url)
      * a timestamp will be added.
      *
      * @param string $path The file path to timestamp, the path must be inside WWW_ROOT
+     * @param bool|string $stamp If set will overrule the value of `Asset.timestamp` in Configure.
      * @return string Path with a timestamp added, or not.
      */
-    public function assetTimestamp($path)
+    public function assetTimestamp($path, $stamp = null)
```

Comment:
Change the argument name `$stamp` to `$timestamp` for consistency?

Reply:
I chose `$stamp` as argument name to keep it consistent with the variable used in the function.

Reply:
You can easily fix up all of them to "timestamp".

Reply:
@JoshuaLuckers Internal variables don't show up in API docs but methods arguments and options do :)